includes/Core/Admin/Screens.php

```diff
@@ -470,7 +470,7 @@ private function get_screens() {
 				array(
 					'title'       => __( 'User Input', 'google-site-kit' ),
 					'capability'  => Permissions::MANAGE_OPTIONS,
-					'parent_slug' => null,
```

Comment:
Since this was defined we also have another instance of this which we may as well address here too: see `ad-blocking-recovery` below.

Also, how about we use a named constant for this slug since its a special case? Maybe something like `Screens::PARENT_SLUG_NULL` ?

Reply:
Thanks @aaemnnosttv this has been resolved in a recent commit.